migrate.py

```diff
@@ -1198,6 +1146,56 @@ def assemble_collections(checkout_path, spec, args, target_github_org):
             REMOVE = set()
 
 
+def galaxy_metadata_init(collection, namespace, target_github_org):
```

Comment:
It's better to start a verb in the function/method name because it represents a process, not something static like a var.
```suggestion
def initialize_galaxy_metadata(collection, namespace, target_github_org):
```
or maybe `generate_` or so